### ISSUE TITLE
[Bug][a11y]: A screen reader should announce a Toast message when it is added to the DOM

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3407]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Toast

### Expected behavior

A screen reader should announce a Toast message when it is added to the DOM. A develop should not need to remember to add `role="status"` or `role="alert"` so that assistive technology will announce the message.

### Actual behavior

A screen reader will not announce a Toast message when it is added to the DOM, without additional developer awareness to add role="status" or role="alert."

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by najikahalsema** [comment]
SWC-281

**Comment by Rajdeepc** [comment]
@majornista This has been addressed on our end. I’ve verified the fix using a screen reader in the Storybook environment:
https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/toast--overlay
Can you please confirm the behavior on your side as well?
